web/docs/tql2/operators/from_file.md

```diff
@@ -0,0 +1,69 @@
+# from_file
+
+:::warning
+This operator is still under active development.
+:::
+
+Reads one or multiple files using from a filesystem.
+
+```tql
+from_file url:string, [watch=bool, remove=bool, path_field=field { … }]
+```
+
+## Description
+
+### `url: string`
```

Comment:
This section should explain that this maps onto Arrow's filesystem (for now), and what URL options exist, e.g., `endpoint_override` for S3.

Reply:
I copy-pasted the things that we have written in the docs for S3 and GCS.